package.json

```diff
@@ -52,6 +53,7 @@
     "prettier": "@baltpeter/prettier-config",
     "dependencies": {
         "@napi-rs/lzma": "^1.1.2",
+        "andromatic": "/home/benni/coding/JS/tweasel/andromatic/andromatic-v0.0.0.tgz",
```

Comment:
Well, this is obviously a TODO, but otherwise I think we can merge this when https://github.com/tweaselORG/andromatic/pull/1 is merged.